addons/l10n_account_withholding/models/res_config_settings.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    l10n_account_withholding_tax_base_account_id = fields.Many2one(
```

Comment:
Are we sure about the name of the module? Isn't `l10n_account_wth/wh` better? It's a code, doesn't have to be 100% clear, it would shorten field/method names a lot.
I think `wh` is better because too many times I've seen `wht` being written by mistake instead of `wth`.

Reply:
> Are we sure about the name of the module
The POs approved it

These are technical names, I understand what you mean. I personally don't mind long names, but I can shorten then module prefix on all of them at least :ok_hand: